editor/plugins/shader_editor_plugin.h

```diff
@@ -84,9 +84,10 @@ class ShaderEditorPlugin : public EditorPlugin {
 		CONTEXT_VALID_ITEM,
 	};
 
-	HSplitContainer *main_split = nullptr;
-	VBoxContainer *left_panel = nullptr;
+	VBoxContainer *main_container = nullptr;
+	HSplitContainer *script_split = nullptr;
```

Comment:
This should probably be renamed `files_split` or something to match the recent rename. https://github.com/godotengine/godot/pull/105124